src/include/kdump/uifunctions.rb

```diff
@@ -263,12 +263,16 @@ def SetUpKDUMP_SAVE_TARGET(target)
       if target != ""
         pos = Builtins.search(parse_target, "/")
         pos1 = -1
-        Ops.set(
-          @KDUMP_SAVE_TARGET,
-          "target",
-          Builtins.substring(parse_target, 0, Ops.subtract(pos, 1))
-        )
-        parse_target = Builtins.substring(parse_target, Ops.add(pos, 2))
+        if pos.zero? # Support for the old '/var/crash' style
+          Ops.set(@KDUMP_SAVE_TARGET, "target", "file")
```

Comment:
I only wanted to fix the code without changing anything else (or mixin 'styles'). IMHO, this function should be rewritten (but not now).